jflex/src/test/java/jflex/core/unicode/IntCharGen.java

```diff
@@ -0,0 +1,171 @@
+package jflex.core.unicode;
+
+import com.pholser.junit.quickcheck.generator.GenerationStatus;
+import com.pholser.junit.quickcheck.generator.Generator;
+import com.pholser.junit.quickcheck.random.SourceOfRandomness;
+
+/**
+ * Generator for random Integer values that ensure to sometimes generate a cased character
+ *
+ * @author Jesse Coultas
+ * @version JFlex 1.8.2
+ */
+public class IntCharGen extends Generator<Integer> {
+  // list of ints that have cased values from Unicode 14.0
+  private static final int[] casedList = {
```

Comment:
There should be a better way to get at these than explicitly listing them for a specific Unicode version. We already have a bunch of classes that represent Unicode versions and encodings. Can we get that information from there?